### PR TITLE
Update cloud-tutorials.md

### PR DESCRIPTION
**Pull request by mageirakos**
Adding a link to https://g.co/codelabs/kfp-gis as requested on issue  #1317

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1730)

### CODE REVIEW

**Comment by mageirakos** [comment]
@googlebot I signed it!

**Comment by sarahmaddox** [comment]
/ok-to-test
/assign @amygdala 

@amygdala I see that the codelab still refers to Kubeflow 0.6.2. I know that you're working on the samples and codelabs, Please would you take a look at this PR, with regard to the current status of the Pipelines GitHub Issue Summarization sample?

File: content/docs/pipelines/tutorials/cloud-tutorials.md [hunk 1]
```diff
@@ -9,4 +9,5 @@ weight = 1
 See the <a href="/docs/about/docs/">guide to the Kubeflow docs</a>.</p>
 {{% /alert %}}
 
-* [Pipelines End-to-end on Azure](/docs/azure/azureendtoend/): An end-to-end tutorial for Kubeflow Pipelines on Microsoft Azure.
\ No newline at end of file
+* [Pipelines End-to-end on Azure](/docs/azure/azureendtoend/): An end-to-end tutorial for Kubeflow Pipelines on Microsoft Azure.
+* [Pipelines - GitHub Issue Summarization](https://g.co/codelabs/kfp-gis) : An end-to-end tutorial for Kubeflow deployment both from  Pipelines UI, and from a Jupyter Notebook.
```
Comment (amygdala) [inline content/docs/pipelines/tutorials/cloud-tutorials.md]: Thanks for doing this. How about a slightly modified blurb that indicates it's a GCP example, and gives a bit more detail?
"This GCP tutorial walks through a Kubeflow Pipelines example that shows training a Tensor2Tensor model for GitHub issue summarization, both via the Pipelines Dashboard UI, and from a Jupyter notebook."

**Comment by amygdala** [comment]
I left a small comment on the blurb text. 
@sarahmaddox  -- yeah, I'm updating the codelab doc now (intend to publish the new version tomorrow).  I ran through it a few weeks ago successfully and so it 'should' all work with KF 1, but I'm going to do one more runthrough with the latest RC. So it should be okay to add the link.

**Comment by amygdala** [comment]
btw, I'm creating a replacement 'intro' codelab that uses Jeremy's new mnist notebook example (replacing the current intro one).  Should be done w/ that by Monday. @sarahmaddox

File: content/docs/pipelines/tutorials/cloud-tutorials.md (hunk 1 above)
Comment (mageirakos) [reply content/docs/pipelines/tutorials/cloud-tutorials.md]: Yes of course, I will add your comments. I just thought more detail was not needed judging from the tutorial description on Azure which was small.

**Comment by mageirakos** [comment]
@amygdala 

Do you want me to also add the replacement 'intro' codelab on Monday when its done? Or should we open a new good-first-issue for future first timers

**Comment by amygdala** [comment]
/lgtm
Thank you for doing this!  Let's make the new intro codelab a separate small PR, as I am not 100% sure yet whether it will use the existing URL or a new one.

**Comment by sarahmaddox** [comment]
Preview:
https://deploy-preview-1730--competent-brattain-de2d6d.netlify.com/docs/pipelines/tutorials/cloud-tutorials/

Thanks @mageirakos!
/approve

**Pull request merged**